website/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "docusaurus": "docusaurus",
+    "postinstall": "pnpm run -C ../ build",
```

Comment:
`tsup` is pretty fast and I think running this on `postinstall` is pretty harmless since these docs need Nuka built to start up and run.

Reply:
Seems reasonable to me.